src/main/java/com/three_tech_solutions/slot_app/data/models/Plan.java

```diff
@@ -12,9 +13,7 @@
 public class Plan {
     private byte classesPerWeek;
     private Byte paymentDay; //porque es opcional
```

Comment:
Este comentario lo podes borrar
```suggestion
    private Byte paymentDay; 
```